### PR TITLE
Add @ logo to pages

### PR DESCRIPTION
**Pull request by Calebasah**

### CODE REVIEW

File: package-lock.json
Comment (stephane-segning) [inline package-lock.json:1]: Use yarn instead and remove this file

**Review by stephane-segning — CHANGES_REQUESTED** [review]
Please can you do a picture of the result?

**Comment by Calebasah** [comment]
> Please can you do a picture of the result?

Screenshot from 2024-07-18 18-10-29

File: package-lock.json
Comment (Calebasah) [reply package-lock.json:1]: Done!
Mr @stephane-segning, are the changes okay?

File: yarn.lock
Comment (stephane-segning) [inline yarn.lock:1]: Please add yarn to this file. Use the command yarn install for that

**Review by stephane-segning — CHANGES_REQUESTED** [review]

**Review by stephane-segning — APPROVED** [review]

**Pull request merged**